src/messages/messages.repository.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from "@nestjs/common";
+
+import { EntityRepository } from "@mikro-orm/postgresql";
+
+import { Message } from "@/common/entities/messages.entity";
+
+@Injectable()
+export class MessageRepository extends EntityRepository<Message> {}
```

Comment:
Same comment as above


```suggestion
export class MessagesRepository extends EntityRepository<Message> {}
```